components/widgets/selectors.js

```diff
@@ -78,7 +78,8 @@ export const selectEmbed = (state, { embed }) => embed;
 export const selectSimple = (state, { simple }) => simple;
 export const selectAnalysis = (state, { analysis }) => analysis;
 export const selectCategory = (state) =>
-  state.location && state.location.query && state.location.query.category;
+  (state.location && state.location.query && state.location.query.category) ||
```

Comment:
"not related", Do we have optional chaining supported in our build flow? would prefer this in the future to clean up these conditions! 

```
export const selectCategory = (state) => state?.location?.query?.category || 'summary';
```

Reply:
Yes we do! It default in babel 7 now!

Reply:
awesome! no need to update now, we can do it as we go along